src/execution/values.ts

```diff
@@ -222,7 +226,7 @@ export function experimentalGetArgumentValues(
         // execution. This is a runtime check to ensure execution does not
         // continue with an invalid argument value.
         throw new GraphQLError(
-          `Argument "${argDef.name}" of required type "${argType}" was not provided.`,
+          `Argument "${isArgument(argDef) ? argDef : argDef.name}" of required type "${argType}" was not provided.`,
```

Comment:
Why would this not be an argument? This check feels a bit redundant given the parent check where we check whether it in fact is a required argument

Reply:
added todo 646d600